extension/ingestionapikeyauthextension/extension.go

```diff
@@ -147,6 +152,7 @@ func checkAuthorizationHeader(authorizationHeader string, exCtx *extensionContex
 		return errAuthServerUnavailable
 	}
 
+	log.Printf("Result for %s: %d\n", authorizationHeader, res.StatusCode)
```

Comment:
:+1:  I was definitely missing this when configuring it for a test and didn't see any data.